### PR TITLE
[SPARK-31565][WEBUI][FOLLOWUP] Add font color setting of DAG-viz for query plan.

### PR DESCRIPTION
**Pull request by sarutak**
### What changes were proposed in this pull request?

This PR adds a font color setting of DAG-viz for query plan.

### Why are the changes needed?

#28352 aimed to unify the font color of all DAG-viz in WebUI but there is one part left over.

Before this change applied, the appearance of a query plan is like as follows.
plan-graph-fixed
The color of `WholeStageCodegen (1)` and its following text (`duration: total...`) is slightly darker than `SerializeFromObject`.

After this change, those color is unified as `#333333`.
plan-graph-fixed2

### Does this PR introduce any user-facing change?

Slightly yes.

### How was this patch tested?

I confirmed the style of `fill` and `color` is `#333333` by debug console of Chrome.
fill
color

### CODE REVIEW

File: sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css [hunk 1]
```diff
@@ -39,6 +39,10 @@
   font-weight: bold;
 }
 
+#plan-viz-graph svg text {
```
Comment (dongjoon-hyun) [inline sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css:42]: Previously, we handled all `.label`. Now, this PR is about `text`.
If we start to consider `text`, are you sure that this is the last `text`?

Comment (sarutak) [reply sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css:42]: I think we need to consider three part of DAG-viz. Stages, RDDs and query plans.

In the previous PR, I changed the color of `.label` for stages and RDDs. Both are covered by [here](https://github.com/apache/spark/pull/28352/files#diff-e5cdb3a11044297da1b3136680150173R22).
I also changed color of `.label` for query plans [here](https://github.com/apache/spark/pull/28352/files#diff-98200c641ee59851dddec016c9c67b7fR22).

About `text`, for stages and RDDs, it's already covered [here](https://github.com/apache/spark/blob/master/core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.css#L75).
So I think the last one is for query plans. It's covered by this PR.

Comment (dongjoon-hyun) [reply sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css:42]: Thanks for the confirmation!

**Comment by dongjoon-hyun** [comment]
Retest this please.

**Comment by dongjoon-hyun** [comment]
@sarutak . I checked the `WholeStageCodegen (1)` is changed from `#000000` to `#333333` correctly. In the same way, the text below `WholeStageCodegen (1)` like the followings are also changed together. If this is intentional, please update the PR description accordingly.
```
duration: total (min, med, max )
545 ms (17 ms, 53 ms, 113 ms )
```

**Comment by sarutak** [comment]
@dongjoon-hyun Thanks for checking. The behavior  you mention is intentional so I've updated the description.

**Comment by sarutak** [comment]
retest this please.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you, @sarutak .
Merged to master/3.0.

**Pull request closed without merging**